app/src/main/AndroidManifest.xml

```diff
@@ -29,7 +29,8 @@
         android:label="@string/app_name"
         android:largeHeap="true"
         android:roundIcon="@mipmap/ic_launcher_round"
-        android:theme="@style/AppTheme.Light">
+        android:theme="@style/AppTheme.Light"
+        android:requestLegacyExternalStorage="true">
```

Comment:
Could you please add a note in the PR as to why this is added?

Reply:
So does this PR fix #101 too? If so could you please add `Fixes #101` to the top comment in PR?